src/core/trulens/core/utils/pyschema.py

```diff
@@ -146,14 +139,14 @@ class Module(SerialModel):
     package_name: Optional[str] = None  # some modules are not in a package
     module_name: str
 
-    def of_module(mod: ModuleType, loadable: bool = False) -> Module:
+    def of_module(mod: ModuleType, loadable: bool = False) -> "Module":
```

Comment:
Is this a change in logic?

Reply:
No just a type hint. Looking at it now, this is actually not necessary.